app/views/customers/show.html.erb

```diff
@@ -0,0 +1,22 @@
+<h1>Customer Name: <%= @customer.name %></h1>
+
+<section class="items">
+<h3>Customer Items: </h3>
+<% @customer.items.each do |item| %>
+  <ul>
+  <li>Item Name: <%= item.name %></li>
+  <li>Item Price: <%= item.price %></li>
+  <li>Item Location: <%= item.supermarket.name %></li>
+  </ul>
+  <% end %>
+</section>
+
+<section class="add_item">
+<h3>Add Item: </h3>
+<%= form_with url: "/customers/#{@customer.id}", method: :post, local: true do |f| %>
```

Comment:
One thing that we could add to this url to create our customer item connection is
```ruby
"/customers/#{customer.id}/items"
```